docs/tutorials/embedding/MASE.ipynb

```diff
@@ -0,0 +1,374 @@
+   "source": [
+    " Although we don't get exactly the same block probabilities as we input into the SBM, it's fairly close with only 1 graph for each!"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Sampling More Graphs"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "We can recover an even more accurate representation of the two SBMs with more sample graphs.  Let's try with 100 graphs sampled from each.  Interestingly, this time ``MultipleASE()`` chooses to embed into 4 dimensions."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "G1 = [sbm(n, P1) for i in range(100)]\n",
+    "G2 = [sbm(n, P2) for i in range(100)]\n",
+    "\n",
+    "V_hat = embedder.fit_transform(G1 + G2)\n",
+    "print(V_hat.shape)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Visualizing Common Latent Subspace"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "As before, the latent subspace shows two distinct clusters, especially in dimensions 1 vs dimension 2."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "from graspy.plot import pairplot\n",
+    "labels = [\"Block 1\"]*25 + [\"Block 2\"]*25\n",
+    "pairplot(V_hat, labels, title=\"Common Latent Subspace from MASE Embedding\")"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Recover Probability Matrices"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Since we have many samples, we first obtain the estimated probability matrix $\\hat{P}^{(i)} = V R^{(i)} V^T$ for each graph, and then we average across all the graphs generated from the same SBM in order to recover the probabiliity matrices."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "#Obtain list of P_hats for each graph\n",
+    "R = embedder.scores_\n",
+    "V_hat_T = np.transpose(V_hat)\n",
+    "P_hat = np.array([np.matmul(np.matmul(V_hat, R[i]), V_hat_T) for i in range(len(R))])\n",
+    "\n",
+    "#Average across all graphs generated from the same distribution.\n",
+    "P1_hat = np.mean(P_hat[:100], 0)\n",
+    "P2_hat = np.mean(P_hat[100:], 0)\n",
+    "heatmap(P1_hat, figsize=(7,7), title=\"Visualization of Probability Matrix 1\")\n",
+    "heatmap(P2_hat, figsize=(7,7), title=\"Visualization of Probability Matrix 2\")"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Recover Original Block Probabilities"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "And we see that our estimates for the block probabilities are closer to the true values with a higher number of samples."
```

Comment:
not really, intrablock probability in the first graph unchanged. i know this is because you got a little 'unlucky' on the sample, but still i'd offer to use even more graphs to really make the point clear.

Reply:
intrablock probability in the first graph second block to be precise

Reply:
this still holds?